pkg/deploy/generator/resources_gateway.go

```diff
@@ -554,6 +554,16 @@ cat >/etc/default/vsa-nodescan-agent.config <<EOF
   }
 EOF
 
+# we start a cron job to run every hour to ensure the said directory is accessible 
+# by the correct user as it gets created by root and may cause a race condition 
+# where root owns the dir instead of syslog
+# TODO: samann rm this once bug is fixed by MSFT
```

Comment:
Do we have anything tracking the bug on the MSFT side?  Ideally we have ADO link here.  If not for MSFT then for us in our backlog.

Reply:
Will update to include our open issue https://github.com/Azure/ARO-RP/issues/1837 which has the MSFT tracking